librad/src/git/storage/fetcher.rs

```diff
@@ -416,7 +430,13 @@ mod imp {
             urn: Urn,
             remote_peer: PeerId,
         ) -> Result<Self, git2::Error> {
-            let mut remote = storage.as_raw().remote_anonymous(url.as_str())?;
+            let mut remote = match storage.as_raw().remote_anonymous(url.as_str()) {
+                Ok(r) => r,
+                Err(e) => {
+                    tracing::error!(?url, error=?e, "Error opening remote");
```

Comment:
nit: `err=?e` is used elsewhere. If/when we decide to employ some kind of log indexing or trace collector, it might come in handy to be able to query for that field.

Reply:
Fix pushed